includes/Core/Assets/Assets.php

```diff
@@ -781,6 +789,39 @@ private function get_inline_dashboard_sharing_data() {
 			}
 		}
 
+		// Expose recoverable modules to the client.
+		$modules           = new Modules( $this->context );
+		$shareable_modules = $modules->get_shareable_modules();
```

Comment:
Instead of creating a new instance of the Modules class here, let's move this logic to a new method `get_recoverable_modules` of the Modules class and add a hook for the `googlesitekit_dashboard_sharing_data` filter to call the new method.

Reply:
That's a good idea 👍